staging/src/k8s.io/metrics/pkg/apis/custom_metrics/install/install.go

```diff
@@ -1,49 +0,0 @@
-/*
```

Comment:
custom metrics doesn't have a generated clientset.  We'll need to do something manually.